meshpy/four_c/dbc_monitor.py

```diff
@@ -36,8 +36,73 @@
 # Python modules.
 import numpy as np
 
+import meshpy
```

Comment:
This looks weird. Can't you use `from meshpy import ...`?

Reply:
I removed it, this was not necessary at all.